templates/base.twig

```diff
@@ -11,7 +11,8 @@
 		<li><a href="#content">{{ __( 'Skip to Content', 'planet4-master-theme' ) }}</a></li>
 		<li><a href="#footer">{{ __( 'Skip to Footer', 'planet4-master-theme' ) }}</a></li>
 	</ul>
-
+	<a style="display: none;" title="{{ __('Go to the top of the page.', 'planet4-master-theme') }}" class="back-top"
```

Comment:
Instead of the inline `style` here you can probably use the `d-none` class [from bootstrap](https://getbootstrap.com/docs/4.5/utilities/display/#notation).